docs/organizations/managing-security-and-risk.md

```diff
@@ -32,29 +32,85 @@ Each of these areas displays a count of matching items and a **Review** button t
 
 The Security and Risk Management item list displays a list of items, filtered by status and sorted by due date ascending.
 
-To access the item list, access the [dashboard](#dashboard) and click the **Review** button under the area of interest, based on the desired filtering.
+To access the item list, access the [dashboard](#dashboard) and click the **Review** button under an area of interest, based on the desired filtering.
 
-To find out more about an item, click its **Details** column. This redirects you to a filtered view of the **Repository issues** page, showing only the item of interest.
+To open the originating issue of an item, click the **Details** column of the item of interest.
```

Comment:
I saw some of [your comments](https://github.com/codacy/sla-service/pull/100#discussion_r1221141447) reviewing the API and I liked the word "underlying". 😉 

Would it work here? 

```suggestion
To open the underlying issue of an item, click the **Details** column of the item of interest.
```